app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.example.ui.home
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class HomeActivity : WolmoActivity() {
+
+    override fun layout(): Int {
+        return R.layout.activity_base
+    }
+
+    override fun init() {
+        replaceFragment(R.id.vActivityBaseContent, HomeFragment.instance)
+    }
+
+    companion object {
+        fun Context.start() {
+            val intent = Intent(this, HomeActivity::class.java)
+            startActivity(intent)
```

Comment:
Wow, this is interesting. I guess this works since you have make the PR. We usually use the start pattern like this:
```
fun start(context: Context) {
val intent = Intent(context, HomeActivity::class.java)
startActivity(intent)

```